test/model/content_test.dart

```diff
@@ -735,14 +735,32 @@ class ContentExample {
         '<img data-original-dimensions="6000x4000" src="/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp"></a></div>', [
     ImagePreviewNodeList([
       ImagePreviewNode(srcUrl: '/user_uploads/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg',
-        thumbnailUrl: '/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp',
+        thumbnail: ImageThumbnailLocator(animated: false,
+          urlPath: '/user_uploads/thumbnail/2/ce/nvoNL2LaZOciwGZ-FYagddtK/image.jpg/840x560.webp'),
         loading: false,
         originalWidth: 6000,
         originalHeight: 4000),
     ]),
   ]);
 
-  static const imagePreviewSingleNoDimensions = ContentExample(
+  static final imagePreviewSingleAnimated = ContentExample(
+    'single image preview, with animated version',
```

Comment:
nit: something like
```suggestion
  static final imagePreviewSingleAnimated = ContentExample(
    'single image preview, animated',
```

since the animated version is the primary one